app/controllers/response_sets_controller.rb

```diff
@@ -79,9 +96,14 @@ def set_response_set
     @response_set = ResponseSet.find(params[:id])
   end
 
+  def set_parent_set
+    @parent_set = ResponseSet.find(params[:id])
+    @parent_id = @parent_set.id
+  end
+
   # Never trust parameters from the scary internet, only allow the white list through.
   def response_set_params
-    params.require(:response_set).permit(:name, :description, :oid, :author, :coded,
+    params.require(:response_set).permit(:name, :description, :parent_id, :oid, :author, :coded,
```

Comment:
Why is parent_id included in params?  This should be set once upon create and not touched again.